### DISCUSSION TITLE
Incoming emails not accepted after. GitHub page CNAME update

### DISCUSSION THREAD

**Discussion by laminjalang** [community/community#37715] [Other Feature Feedback, Questions, & Ideas]
I recently pointed my custom domain to GitHub pages but this is preventing me from receiving emails on my domain.
Any help will be really appreciated.

**Comment by airtower-luna** [comment]
Make sure you have a correct MX record for your domain that points at your mail server. Mind that the mail server can't have the same hostname as your pages site, because they'll have different IP addresses.

**Reply by laminjalang** [reply]
Thank you very much airtower-luna.
In that case do I just make up a different hostname from my domain name?

**Reply by airtower-luna** [reply]
Sure, any hostname you actually have available. Make sure to create the corresponding A/AAAA records first though. A name that can't be resolved won't help. :wink: 

Oh, and make sure the mail server is aware of the change, too, some other servers may not want to talk to it if the HELO/EHLO response doesn't fit the hostname, or if the certificate (I hope you use TLS) doesn't fit. Basically, check the whole mail server setup, and all related DNS entries.
